strongbox-web-core/pom.xml

```diff
@@ -306,7 +306,7 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
-                    <mainClass>org.carlspring.strongbox.StrongboxSpringBootApplication</mainClass>
+                    <mainClass>org.carlspring.strongbox.app.StrongboxSpringBootApplication</mainClass>
```

Comment:
What's the magic spell you cast here? I tried having this in a different package and it didn't seem to work... @fuss86 advised to place it in the root package and it started working. I was thinking of adding it under the `org.carlspring.strongbox.booters` package.

Reply:
no magic, I just renamed it, that's all :)